package.json

```diff
@@ -44,6 +44,7 @@
 	},
 	"type": "module",
 	"dependencies": {
+		"cross-fetch": "^4.0.0",
```

Comment:
please explain your research and reasoning behind why this is required - correct me if the linked webpage below is insufficient/incorrect 

https://developer.mozilla.org/en-US/docs/Web/API/fetch

Reply:
It is needed to run tests because fetch is not native to the Node.js enviornent.